Geometry/VeryForwardGeometry/data/dd4hep/v5/geometryRPFromDD_2021.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>
```

Comment:
The only change in this new version is the removal of the `ppstrackerMaterials.xml` file.